components/json/rs/src/lib.rs

```diff
@@ -159,3 +164,51 @@ pub fn convert_string_value_to_int64(path: &str, json: &str, optional: bool) ->
 
     String::new()
 }
+
+// Parses and re-serializes json with uint64 values for the given key in the
+// object array at path are converted to a string.
+// Original json string will be returned if object array at path is not found.
+// When the target uint64 value at key is not found in an object or is null,
+// the value will be unchanged.
+// Examples:
+// convert_uint64_in_object_array_to_string("/a","key", json) for
+// {a: [{"key": 1}, {"key": 2}, {"key": null]} -> {a: [{"key": "1"}, {"key": "2"}, {"key": null}]}
+pub fn convert_uint64_in_object_array_to_string(path: &str, key: &str, json: &str) -> String {
+    let result_value = serde_json::from_str(&json);
+    if result_value.is_err() {
+        return String::new();
+    }
+    let mut unwrapped_value: serde_json::Value = result_value.unwrap();
+    let mutable_pointer = unwrapped_value.pointer_mut(path);
+    if mutable_pointer.is_none() {
+        return json.to_string();
+    }
+
+    let objects = mutable_pointer.unwrap();
+    if !objects.is_array() {
+        return String::new();
+    }
+
+    let unwrapped_objects = objects.as_array_mut().unwrap();
+    for object in unwrapped_objects {
```

Comment:
you can do this here without unwrap and without checking is_array in advance
```rust
if Some(objects) = mutable_pointer.unwrap().as_array_mut() {
  for object in objects {
  ...
} else {
  return String::new();
}
```

Reply:
the ` mutable_pointer.unwrap()` can get from what @antonok-edm suggest without this unwrap

Reply:
https://github.com/brave/brave-core/pull/13033/commits/4015a2e4bafbd6cd846d4d050187dd0807bc4684